Hi3Helper.Core/Lang/zh_CN.json

```diff
@@ -502,7 +513,7 @@
   },
 
   "_Dialogs": {
-    "DeltaPatchDetectedTitle": "警告。检测到Delta补丁更新！",
+    "DeltaPatchDetectedTitle": "警告：检测到 Delta 补丁更新！",
```

Comment:
在 **89** 行，我们把Delta-Patch完整保留了，我觉得之前在Transifex上留下的问题仍然使用，我们是否应该将其作为一个专有名词处理。

In line **89**, we keep "Delta-Patch" as is. I think the question left on Transifex is still meaningful, whether we should treat it as a special name.

![图片](https://github.com/CollapseLauncher/Collapse/assets/42690037/8e71fb5b-f2a2-44ab-adb0-4a6efa59adb3)

Reply:
看起来是疏漏了

Reply:
> 看起来是疏漏了

如果可以的话建议检查一下我留了哪些评论？（老实说，我忘记我在哪些下面评论过了，非常抱歉）（因为我目前所使用的电脑打开Transifex特别缓慢，辛苦您了）
If possible, would you fully checking how many comments I have left? (Honestly, I forgot which ones I commented on, so sorry) (Because the computer I am currently using is very slow to open Transifex, thank you for your hard work)